zhmccli/_cmd_character_rule.py

```diff
@@ -197,7 +197,6 @@ def cmd_character_rule_list(cmd_ctx, password_rule_name):
         'numeric',
         'special',
         'custom-character-sets',
-        'description',
```

Comment:
I just saw the data model for this character rule object, I couldn't find this property `"index"`, Is it added for some purpose ?

Reply:
![image](https://github.com/zhmcclient/zhmccli/assets/58299461/ba454658-bde4-44b4-bad4-276589c5c652)